chimney/src/main/scala/io/scalaland/chimney/ErrorPathNode.scala

```diff
@@ -0,0 +1,33 @@
+package io.scalaland.chimney
+
+sealed trait ErrorPathNode {
+  def show: String
+
+  def separator: String
+}
+
+object ErrorPathNode {
+  case class Accessor(name: String) extends ErrorPathNode {
+    def show: String = name
+
+    val separator: String = "."
```

Comment:
Suggest making it `def`, as it will be materialized eagerly even if one doesn't invoke pretty-printing paths.

Alternatively, instead of both fields you may define separate functions in companion object.

Reply:
Changed to def